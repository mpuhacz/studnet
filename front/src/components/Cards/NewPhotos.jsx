import React from 'react';

const NewFriend = ({ name, avatar, likes, photos = []}) => (
  <div className="event">
    <div className="label">
      <img src={avatar} alt="avatar"/>
    </div>
    <div className="content">
      <div className="summary">
        <a>{name}</a> dodał <a>{photos.length} nowe zdjęcia</a>
        <div className="date">
          4 dni temu
        </div>
      </div>
      <div className="extra images">
      {
        photos.map((photo, index) => (
          <a href={photo} target="_blank" key={index}>
            <img src={photo} alt="Illustration" />
          </a>
        ))
      }
      </div>
      <div className="meta">
        <a className="like">
          <i className="like icon"></i> {likes} Like
        </a>
      </div>
    </div>
  </div>
);

export default NewFriend;
