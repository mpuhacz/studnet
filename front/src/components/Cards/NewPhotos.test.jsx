import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import NewPhotos from './NewPhotos';

describe('<NewPhotos />', () => {
  it('renders NewPhotos without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<NewPhotos
      avatar="https://semantic-ui.com/images/avatar/small/justen.jpg"
      name="Name"
      likes={1}
      photos={[
        'http://www.readersdigest.ca/wp-content/uploads/2011/01/4-ways-cheer-up-depressed-cat.jpg',
        'http://www.elonka.com/kryptos/sanborn/KGBEnglish.jpg',
      ]}
    />, div);
  });
});
