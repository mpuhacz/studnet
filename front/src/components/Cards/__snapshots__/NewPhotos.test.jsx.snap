exports[`<NewPhotos /> renders NewPhotos form with correct specification 1`] = `
<div
  className="event">
  <div
    className="label">
    <img
      alt="avatar"
      src="https://semantic-ui.com/images/avatar/small/justen.jpg" />
  </div>
  <div
    className="content">
    <div
      className="summary">
      <a>
        Name
      </a>
       dodał 
      <a>
        2
         nowe zdjęcia
      </a>
      <div
        className="date">
        4 dni temu
      </div>
    </div>
    <div
      className="extra images">
      <a
        href="http://www.readersdigest.ca/wp-content/uploads/2011/01/4-ways-cheer-up-depressed-cat.jpg"
        target="_blank">
        <img
          alt="Illustration"
          src="http://www.readersdigest.ca/wp-content/uploads/2011/01/4-ways-cheer-up-depressed-cat.jpg" />
      </a>
      <a
        href="http://www.elonka.com/kryptos/sanborn/KGBEnglish.jpg"
        target="_blank">
        <img
          alt="Illustration"
          src="http://www.elonka.com/kryptos/sanborn/KGBEnglish.jpg" />
      </a>
    </div>
    <div
      className="meta">
      <a
        className="like">
        <i
          className="like icon" />
         
        1
         Like
      </a>
    </div>
  </div>
</div>
`;
